mapsforge-map-awt/src/main/java/org/mapsforge/map/awt/graphics/AwtPaint.java

```diff
@@ -171,6 +185,45 @@ public int getTextWidth(String text) {
         return fontMetrics.stringWidth(text);
     }
 
+    @Override
+    public int getTextWidth(String text, boolean includePadding) {
+        Graphics2D graphics2d = bufferedImage.createGraphics();
+        FontMetrics fontMetrics = graphics2d.getFontMetrics(this.font);
+        graphics2d.dispose();
+        int retVal = fontMetrics.stringWidth(text);
+
+        if (includePadding && retVal > 0) {
+            retVal += getFontPadding(fontMetrics);
+        }
+
+        return retVal;
+    }
+
+    @Override
+    public int getTextWidth(final String text, final int widthMax, boolean includePadding) {
+        return getTextWidth(text, includePadding);
```

Comment:
Here we can use code from `AwtPointTextContainer` to measure the width instead of default.

Like we use in `AndroidPaint` code from `AndroidPointTextContainer`

Reply:
It's probably easier to just remove these three-argument getTextWidth methods, not used anyway.

Reply:
> It's probably easier to just remove these three-argument getTextWidth methods, not used anyway.

@Sublimis Better do this. :+1: